### PR TITLE
Updated yaml

### PR DESCRIPTION
**Pull request by olakunlekuye2**

### CODE REVIEW

**Review by prashanthmamidi — DISMISSED** [review]

File: resources/public/api/conf/2.0/application.yaml [hunk 1]
```diff
@@ -133,6 +133,13 @@ paths:
           schema:
             type: string
             example: GB1234567890
+        - in: query
+          name: movementReferenceNumber
+          description: This is an optional query parameter that allows a list of departures and or arrivals to be filtered by an MRN.
```
Comment (drnaylor-oc) [inline resources/public/api/conf/2.0/application.yaml:138]: As this is on the departures endpoint, this one should only mention departures.

File: resources/public/api/conf/2.0/application.yaml [hunk 2]
```diff
@@ -808,6 +815,13 @@ paths:
...
```
Comment (drnaylor-oc) [inline resources/public/api/conf/2.0/application.yaml:820]: As this is on the arrivals endpoint, this one should only mention arrivals.

Comment (olakunlekuye2) [reply resources/public/api/conf/2.0/application.yaml:820]: Done

**Review by drnaylor-oc — COMMENTED** [review]
The files:

* build 2.properties
* plugins 2.sbt
* respository 2.yaml

shouldn't be here.

**Review by drnaylor-oc — APPROVED** [review]

**Pull request merged**